designs/developer/add-45/Pipelines-update/Pipelines-update.md

```diff
@@ -0,0 +1,53 @@
+---
+parent: Developer
+version: 4.5
+---
+# Add page- Updates
+
+Updates to the Add page include switching the order of the resources and pipelines section so resources come before pipelines.
+One more subtle update to the Add page is that the ‘Show pipeline visualization’ will stay open even if the user selects a new builder image or resource during their session.
+
+## Upload from Git- Updates
+
+![image](https://github.com/lwrigh/openshift-origin-design/blob/master/designs/developer/add-45/Pipelines-update/img/import-from-git-1.png)
```

Comment:
```suggestion
![image](img/import-from-git-1.png)
```

Reply:
I think you'll need to update the path to your images like the suggestion above ^^

Reply:
I'll update all of the image paths to be the shortened version.